stacks-signer/src/cli.rs

```diff
@@ -95,7 +96,7 @@ pub struct PutChunkArgs {
     pub slot_version: u32,
     /// The data to upload
     #[arg(required = false, value_parser = parse_data)]
-    pub data: Vec<u8>,
+    pub data: alloc::vec::Vec<u8>,
```

Comment:
Note this weirdness is due to the following: 
https://github.com/clap-rs/clap/discussions/4695